packages/vscode/src/extension.ts

```diff
@@ -217,9 +174,10 @@ export async function activate(context: ExtensionContext): Promise<void> {
   )
 
   if (!isDebugOrTest) {
-    telemetry.sendEvent('activated', {
-      signature: await telemetry.getSignature(),
-    })
+    // eslint-disable-next-line
+    const extensionId = 'prisma.' + packageJson.name
+    // eslint-disable-next-line
+    const extensionVersion = packageJson.version
```

Comment:
Were these variables missing before?

Reply:
no, I just moved them below, closer to the checkpoint-client call. Before these were used for the AppInsights telemetry as well.